mix.exs

```diff
@@ -18,15 +18,13 @@ defmodule UAParser.Mixfile do
 
   def application do
     [
-      applications: [:logger, :yamerl],
-      env: UAParser.Mixfile.env(),
-      mod: {UAParser, []},
+      applications: [:logger, :yomel]
     ]
   end
 
   defp deps do
     [
-      {:yamerl, "~> 0.4.0"},
+      {:yomel, "~> 0.5.0"},
```

Comment:
There's no need for this package.  In fact, we _shouldn't_ encourage people to take Erlang packages, wrap them in Elixir code, and call it something new.